examples/svm/plot_rbf_parameters.py

```diff
@@ -184,14 +183,9 @@ def __call__(self, value, clip=None):
 # interesting range while not brutally collapsing all the low score values to
 # the same color.
 
-plt.figure(figsize=(8, 6))
+plt.figure(figsize=(10, 10))
 plt.subplots_adjust(left=.2, right=0.95, bottom=0.15, top=0.95)
-plt.imshow(scores, interpolation='nearest', cmap=plt.cm.hot,
-           norm=MidpointNormalize(vmin=0.2, midpoint=0.92))
-plt.xlabel('gamma')
-plt.ylabel('C')
-plt.colorbar()
-plt.xticks(np.arange(len(gamma_range)), gamma_range, rotation=45)
-plt.yticks(np.arange(len(C_range)), C_range)
-plt.title('Validation accuracy')
+plot_gridsearch_results(grid.cv_results_, title="Validation accuracy",
+                        cmap=plt.cm.hot,
+                        norm=MidpointNormalize(vmin=0.2, midpoint=0.92))
```

Comment:
You could use plt.tight_layout() here, to prevent x-labels to be cropped out of the [figure](https://12791-843222-gh.circle-artifacts.com/0/home/ubuntu/scikit-learn/doc/_build/html/stable/_images/sphx_glr_plot_rbf_parameters_003.png).